blueprints/ember-frost-core/index.js

```diff
@@ -0,0 +1,41 @@
+module.exports = {
+  afterInstall: function () {
+    const packagesToRemove = [
+      'ember-frost-button',
+      'ember-frost-checkbox',
+      'ember-frost-css-core',
+      'ember-frost-icons',
+      'ember-frost-link',
+      'ember-frost-loading',
+      'ember-frost-password',
+      'ember-frost-select',
+      'ember-frost-text',
+      'ember-frost-textare',
+      'ember-frost-theme'
+    ].map((packageName) => {
+      return {name: packageName}
+    })
+
+    return this.removePackagesFromProject(packagesToRemove)
+      .then(() => {
+        return this.addBowerPackagesToProject([
+          {name: 'lodash', target: '^4.10.0'}
+        ])
+      })
+      .then(() => {
+        return this.addAddonsToProject({
+          packages: [
+            {name: 'ember-computed-decorators', target: '>=0.2.2 <2.0.0'},
+            {name: 'ember-lodash', target: '>=0.0.6 <2.0.0'},
```

Comment:
Does this handle adding lodash in bower/npm?  If so, can we remove the addBower section above?